python/rikai/contrib/torchhub/pytorch/vision/__init__.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from . import resnet
+
+resnet18 = resnet
```

Comment:
where/how are these used? why do they all get aliased to the same thing?

Reply:
because resnet is the impl of pre/post processors for all resnet{n}
and to make the `create model` syntax as simple as enough, there is a default behavior to find the pre/post processing if any